documentation/src/docs/pravega-write-read-methods.md

```diff
@@ -0,0 +1,57 @@
+[Presto & Trino](https://github.com/pravega/presto-connector)                                   | No    | No                  | Yes  |                  |
+[Pravega Java API](javadoc)                                                                     | Yes   | Yes                 | Yes  |                  |
+[Pravega Rust API](https://pravega.github.io/pravega-client-rust/)                              | Yes   | Yes                 | Yes  |                  |
+[Pravega Python API](https://pravega.github.io/pravega-client-rust/Python/index.html)           | Yes   | Yes                 | Yes  |                  |
+[Debezium](https://debezium.io/documentation/reference/operations/debezium-server.html#_pravega)| Yes   | Yes                 | No   |                  |
+[ZIO](https://github.com/cheleb/zio-pravega)                                                    | Yes   | No                  | Yes  |                  |
+
+**Write**: *Yes* means that the method can write data to Pravega streams. Writes must always append to the tail of a Pravega stream.
+
+**Transactional Write**: *Yes* means that the method can write data to Pravega streams using Pravega [transactions](transactions.md). When properly used throughout a stream processing pipeline, this can provide exactly-once semantics.
+
+**Read**: *Yes* means that the method can read data from Pravega streams.
+
+**Special Feature - EO (Exactly-Once)**: Exactly-once semantics means that events are never lost, duplicated, or processed out-of-order. In general, a stream processing pipeline can only offer exactly-once semantics if all components offer exactly-once semantics. This is usually implemented by using [transactional writes](transactions.md) and [Stream Cuts](streamcuts.md).
+
+## Pravega Sensor Collector
+
+[Pravega Sensor Collector](https://github.com/pravega/pravega-sensor-collector) is an application that collects data from sensors and ingests the data into Pravega streams.
+
+Pravega Sensor Collector can continuously collect high-resolution samples without interruption, even if the network connection to the Pravega server is unavailable for long periods. For instance, in a connected train use case, there may be long periods of time between cities where there is no network access. During this time, Pravega Sensor Collector will store collected sensor data on a local disk and periodically attempt to reconnect to the Pravega server. It stores this sensor data in local SQLite database files. When transferring samples from a SQLite database file to Pravega, it coordinates a SQLite transaction and a Pravega transaction. This technique allows it to guarantee that all samples are sent in order, without gaps, and without duplicates, even in the presence of computer, network, and power failures.
+
+<IfHaveFeature feature="nautilus">
+## SDP Ingest Gateway
```

Comment:
Do we need this given that we are just linking the connectors to their repos?

Reply:
You mean why does Pravega Sensor Collector get a description and not other connectors? @claudiofahey provided this page, but perhaps we can clean it up further.